src/components/widgets/Companions/Companions.tsx

```diff
@@ -38,17 +51,41 @@ export const Companions: FC = () => {
             </div>
 
             <FilteredList className={classes.filteredList}>
-                {data?.findUsers.length === 0 ? (
+                {called && findUsers?.length === 0 ? (
                     <Span title={"По вашему запросу ничего не найдено"} />
                 ) : (
-                    data?.findUsers.map((card: any) => (
+                    findUsers.map((card: any) => (
                         <div key={card._id}>
                             <Avatar name={card.name} image={card.image} scale={1.8} id={card._id} />
                             <Span title={card.name} />
+                            <Button resetDefaultStyles onClick={() => companionRequest(card._id)}>
+                                <Plus className={classes.addCompanion} />
+                            </Button>
                         </div>
                     ))
                 )}
             </FilteredList>
+
+            <div className={classes.companions}>
+                <div className={classes.header}>
+                    <Title tag={"h3"} title="My companions" />
+                </div>
+
+                <div className={classes.line}></div>
+
+                <FilteredList className={classes.companionsList}>
+                    {companions &&
+                        companions.map((card: any) => (
```

Comment:
```suggestion
                        companions?.map((card: any) => (
```

и тогда `companions &&` можно убрать

Reply:
исправлено